FirebaseInAppMessaging/Sources/RenderingObjects/FIRInAppMessagingRenderingDataClasses.m

```diff
@@ -27,9 +27,11 @@ - (instancetype)initWithMessageID:(NSString *)messageID
                       triggerType:(FIRInAppMessagingDisplayTriggerType)triggerType
                           appData:(NSDictionary *)appData {
   if (self = [super init]) {
+    #pragma clang diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
Add `#pragma clang diagnostic push`

Run scripts/style.sh 
I believe pragma's should be in column 1.